azurelinuxagent/common/protocol/wire.py

```diff
@@ -59,16 +61,24 @@
 P7M_FILE_NAME = "Certificates.p7m"
 PEM_FILE_NAME = "Certificates.pem"
 EXT_CONF_FILE_NAME = "ExtensionsConfig.{0}.xml"
+EXT_CONFIG_FAST_TRACK_FILE_NAME = "FastTrackExtensionsConfig.{0}.xml"
 MANIFEST_FILE_NAME = "{0}.{1}.manifest.xml"
 AGENTS_MANIFEST_FILE_NAME = "{0}.{1}.agentsManifest"
 TRANSPORT_CERT_FILE_NAME = "TransportCert.pem"
 TRANSPORT_PRV_FILE_NAME = "TransportPrivate.pem"
 # Store the last retrieved container id as an environment variable to be shared between threads for telemetry purposes
 CONTAINER_ID_ENV_VARIABLE = "AZURE_GUEST_AGENT_CONTAINER_ID"
 
+FAST_TRACK_EXTENSIONS_PATTERN = re.compile("^(FastTrackExtensionsConfig)\.(\d+)\.(xml)$", re.IGNORECASE)
```

Comment:
let's make this a raw string (add r as a prefix):
```
r"^(FastTrackExtensionsConfig)\.(\d+)\.(xml)$"
```

Reply:
Changed